vertx-spring-boot-starter-kafka/pom.xml

```diff
@@ -34,12 +34,20 @@
           <groupId>io.vertx</groupId>
           <artifactId>vertx-kafka-client</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
Why do we exclude and explicitly add zookeeper at the same time?

Reply:
Because Smallrye and Spring Boot Kafka test dependencies are using different zookeeper dependencies which make the tests fail with a not found class exception.
See https://github.com/snowdrop/vertx-spring-boot/pull/106/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R47

Reply:
I was about to say that I would rather solve this via `dependencyManagement` however, using `dependencyManagement` with scope seems to be a bit controversial so, let's keep it as is.